addons/sale_timesheet/tests/test_product_template.py

```diff
@@ -0,0 +1,39 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.exceptions import ValidationError
+from odoo.addons.sale_timesheet.tests.common import TestCommonSaleTimesheet
+from odoo.tests import tagged
+
+
+@tagged('-at_install', 'post_install')
+@tagged('test_product_template')
```

Comment:
```suggestion
```

Not sure what do you want to do, here ?

Reply:
That was for my test automation on my machine, I removed it!